packages/block-editor/src/components/link-control/README.md

```diff
@@ -281,6 +281,7 @@ If passed, children are rendered after the input.
 			label={ __( 'Submit' ) }
 			icon={ keyboardReturn }
 			className="block-editor-link-control__search-submit"
+			size="compact"
```

Comment:
Since the code example focuses on `LinkControlSearchInput`, I believe this prop on the `Button` component is unnecessary.

Reply:
I'm not sure I agree. Your argumentation could then be made for other things in this example, for example the `HStack` component would be unnecessary together with the `justify` prop. The `className` prop would be unnecessary. Instead, I think that code examples should be a full representation of the intended usage because developers will just copy and paste them. If the code example is incomplete or inaccurate, that would lead to inconsistencies or even bugs.

Reply:
I see. I don't have a strong opinion on this, so either is fine. However, I was just curious because there was no code example of opting in this prop to the `Button` component in the Gutenberg documentation.